src/components/custom/_multi-width-column-download.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Header } from 'components/elements/typography'
+import { LocalizedLink, localize } from 'components/localization'
+import Flex from 'components/containers/flex'
+import device from 'themes/device'
+
+type TItem = {
+    text: string
+    icon: string
+    link: string
+    smallText?: string
+}
+
+type TProps = {
+    items?: TItem[]
+    QRImage?: string
+    QRHeading1?: string
+    QRHeading2?: string
+    is_rtl: boolean
+}
+
+const DownloadAppWrapper = styled.div<{ is_rtl: boolean }>`
+    max-width: 384px;
```

Comment:
Aren't we using rem for width?

Reply:
It's deprecated copy which should be replaced once done with component